src/Draft/MyTeam/MyTeam.js

```diff
@@ -18,7 +18,7 @@ class MyTeam extends Component {
   componentDidMount() {
     this.subscription = new Subscription('/teamChat',
     (newChat) => {
-      this.setState({chat: this.state.chat.push(newChat)})
+      this.setState({chat: [...this.state.chat, newChat]})
```

Comment:
Is this a style change or does mutability matter?

Reply:
Its for the purpose of mutability.  React can't diff the two states if you use push.

Reply:
That makes sense.